src/cpu/x64/jit_uni_dw_conv_row_f32.cpp

```diff
@@ -0,0 +1,673 @@
+/*******************************************************************************
+* Copyright 2019-2020 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#include "common/c_types_map.hpp"
+#include "common/dnnl_thread.hpp"
+#include "common/nstl.hpp"
+#include "common/utils.hpp"
+
+#include "cpu/x64/jit_uni_dw_conv_row_f32.hpp"
+
+namespace dnnl {
+namespace impl {
+namespace cpu {
+namespace x64 {
+
+using namespace Xbyak;
+using namespace dnnl::impl::utils;
+
+#define GET_OFF_DW(field) offsetof(jit_conv_call_s, field)
+
+template <cpu_isa_t isa>
+void jit_uni_dw_conv_row_f32<isa>::load_src(int ur_w) {
```

Comment:
Are we sure that the name of the method corresponds to its action? It seems that it just nullifies a number of registers.

Reply:
Agree, I renamed it to `clear_registers()`. Thanks